tests/test_transform.py

```diff
@@ -131,5 +132,46 @@ def testCreatingQuantumGraphMixed(self):
         assert final.queue == "special_final_queue"
 
 
+class TestGetJobValues(unittest.TestCase):
+    """Tests of _get_job_values."""
+
+    def setUp(self):
+        self.gwjob = GenericWorkflowJob("default_job")
+
+    def testGettingDefaults(self):
+        """Test retrieving default values."""
+        config = BpsConfig({})
+        job_values = _get_job_values(config, {}, None)
+        assert all(
+            [
+                getattr(self.gwjob, field.name) == job_values[field.name]
+                for field in dataclasses.fields(self.gwjob)
+                if field.name in job_values
+            ]
+        )
+
+    def testEnablingMemoryScaling(self):
+        """Test enabling the memory scaling mechanism."""
+        config = BpsConfig({"memoryMultiplier": 2.0})
+        job_values = _get_job_values(config, {}, None)
+        self.assertAlmostEqual(job_values["memory_multiplier"], 2.0)
+        assert job_values["number_of_retries"] == 5
```

Comment:
This could be a similar assert all (except memory_multiplier) or is number_of_retries set to 5 in _get_job_values() only if memoryMultiplier is set?

Reply:
Yes, `number_of_retires` is only set when `memory_multiplier` is set to a correct value (> 1).